scripts/build_ses.sh

```diff
@@ -0,0 +1,284 @@
+#!/bin/bash
+# for ses5 install instructions on which this was based see: http://beta.suse.com/private/SUSE-Storage-Beta/doc/ses-manual_en/ceph.install.saltstack.html
+NUM_MASTER=1
+NUM_ADMIN=1
+NUM_MON=3
+NUM_MDS=1
+NUM_IGW=0
+NUM_RGW=2
+
+ip_cidr_regex='^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/([0-9]|[1-2][0-9]|3[0-2]))$'
+
+if [ $# -lt 2 ]
```

Comment:
nit: please use either `[` or `[[` to make its use consistent. My personal preference is `[[` since that's bash internal. And while you are at it, in this particular case you could even use
```
if (( $# < 2 ))
```
:smile: